serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -181,9 +181,33 @@ export class _GLTFMaterialExporter {
             }
         }
 
+        // Do a naive check for materials that contain extensions that have texture-related members.
+        if (material.extensions) {
+            for (let extension in material.extensions){

```

Comment:
nit:
```suggestion
            for (let extension in material.extensions) {
```

Might consider formatting the file.